### DISCUSSION TITLE
Rime能不能解决一个Windows版PS的顽疾

### DISCUSSION THREAD

**Discussion by Aiden-1124** [rime/home#1732] [Q&A]
最近刚换上Rime，很喜欢，也在学习中。。。看到高手通过代码配置自己的输入法，突然想到一个困扰已久的问题：
PS（Windows版）在非输入状态且为中文模式下是没办法使用快捷键的，必须要切换为英文才行，很恼火
所以想请教一下有没有可能通过个性化配置解决这个问题？比如在非键入状态下（无输入光标）就自动切换为英文状态。。。
![qpDmMoHmCL](https://github.com/user-attachments/assets/1a1ac7b8-ed4e-49be-87e5-213d4cd0e147)
当然这应该是Adobe的问题，很多Win软件在中文状态下也是能正常使用快捷键的，但Mac版PS又没问题，无语😮‍💨

**Comment by yfdyh000** [comment]
您可以考虑默认英文模式来部分解决？
```
weasel.custom.yaml
patch:
  "app_options/cmd.exe":
    ascii_mode: true
```
